controls/datagrid/aggregates/styling.md

```diff
@@ -12,7 +12,31 @@ The .NET MAUI DataGrid provides a styling functionality for its group footer, he
 
 ## Styling the Group Header
 
-The DataGrid provides the `GroupHeaderStyle` (`DataGridGroupHeaderStyle`) configuration which defines the style of the `GroupHeader` and the aggregates inside the header.
+The DataGrid provides the `GroupHeaderStyle` (`DataGridGroupHeaderStyle`) configuration which defines the style of the `GroupHeader` and the aggregates inside the header. 
+
+You can style the aggregates with the current properties:
+
+* `AggregatesTextColor`&mdash;Defines the color for the aggregates part of the `GroupHeader`.
+* `AggregatesTextFontAttributes`&mdash;Defines the font attributes for the aggregates part of the `GroupHeader`.
+* `AggregatesTextFontFamily`&mdash;Defines the font family of the aggregates part of the `GroupHeader`.
+* `AggregatesTextFontSize`&mdash;Defines the size of the aggregates part of the `GroupHeader`.
+* `AggregatesTextMargin`&mdash;Defines the margin for the aggregates part of the `GroupHeader`.
+
+And you can style the rest part of the group header with the current properties:
```

Comment:
Style the group header by using the following properties:

This should be in a table as the properties are more than 6